### DISCUSSION TITLE
R9mm won't bind

### DISCUSSION THREAD

**Discussion by Saasi-FPV** [ExpressLRS/ExpressLRS#460] [Q&A]
Hello, I'm new to ELRS. 
I started with flashing the bootloader on the TX (R9M2019) RX (R9mm) all went successful. Then I flashed the TX via the configurator Build firmware.elrs. With the same settings for the R9mm, I flashed that one as well. With no errors, all went as described in the forums. But still, the RX won't Bind to the TX. The RX just flashes RED every second(see Attached files). I tried reflashing both RX and TX with different settings but still couldn't get it to bind.
I'm really out of ideas now. I hope you could help me troubleshoot it. :)

https://user-images.githubusercontent.com/82196317/114319482-0923f480-9b12-11eb-8a9e-a8e05e44e1a1.mp4

**Comment by gregofpv** [comment]
hello friend I have the same problem you managed to solve it

**Reply by KhAnh05101999** [reply]
ave you solved it yet?

**Comment by KhAnh05101999** [comment]
> Xin chào, tôi mới sử dụng ELRS. Tôi đã bắt đầu với việc cài đặt bộ nạp khởi động trên TX (R9M2019) RX (R9mm), tất cả đều thành công. Sau đó, tôi flash TX thông qua cấu hình Build firmware.elrs. Với các cài đặt tương tự cho R9mm, tôi cũng đã flash cái đó. Không có lỗi, tất cả đã diễn ra như mô tả trong diễn đàn. Tuy nhiên, RX sẽ không ràng buộc với TX. RX chỉ nhấp nháy ĐỎ mỗi giây (xem Tệp đính kèm). Tôi đã thử làm mới cả RX và TX với các cài đặt khác nhau nhưng vẫn không thể liên kết được. Tôi thực sự hết ý tưởng bây giờ. Tôi hy vọng bạn có thể giúp tôi gỡ rối nó. :)
> 
>  VID_20210411_203050.mp4

have you solved it yet?

**Comment by radsonpatrick** [comment]
i`m have the same problem !
